www/include/monitoring/objectDetails/xml/hostSendCommand.php

```diff
@@ -49,39 +50,55 @@
 }
 $pearDB = new CentreonDB();
 $hostObj = new CentreonHost($pearDB);
-$host_id = $_GET["host_id"];
-$poller = $hostObj->getHostPollerId($host_id);
-$cmd = $_GET["cmd"];
-$sid = session_id();
-$act_type = $_GET["actiontype"];
+
+$hostId = filter_var(
+    $_GET['host_id'] ?? false,
+    FILTER_VALIDATE_INT
+);
+
+$pollerId = $hostObj->getHostPollerId($hostId);
+
+$cmd = $_GET['cmd'];
+$actionType = $_GET['actiontype'];
```

Comment:
```suggestion
$actionType = (int)$_GET['actiontype'];
```
to be consistent with: https://github.com/centreon/centreon/pull/8803/files#diff-efd752f1863e94d397eb186321af7e7bR95

Reply:
**Done** Thanks :+1: